src/Raven.Studio/typescript/models/database/index/configurationItem.ts

```diff
@@ -31,6 +31,10 @@ class configurationItem {
         "Indexing.Throttling.TimeIntervalInMs",
         "Indexing.TimeSinceLastQueryAfterWhichDeepCleanupCanBeExecutedInMin",
         "Indexing.TransactionSizeLimitInMb",
+        "Indexing.OrderByScoreAutomaticallyWhenBoostingIsInvolved",
+        "Indexing.Lucene.UseCompoundFileInMerging",
+        "Indexing.UseCompoundFileInMerging",
```

Comment:
It should be `Indexing.Lucene.UseCompoundFileInMerging`

Reply:
Done